api/http/api_useradm.go

```diff
@@ -586,6 +586,9 @@ func (u *UserAdmApiHandlers) GetSettingsHandler(w rest.ResponseWriter, r *rest.R
 
 	settings, err := u.db.GetSettings(ctx)
 
+	// remove internal "tenant_id" field from the result
+	delete(settings, "tenant_id")
```

Comment:
@merlin-northern I misunderstood your comment about tenant_id field in the settings; I fixed the issue by explicitly removing the field from settings map before returning settings object